lib/plugins/cannulaage.js

```diff
@@ -3,53 +3,110 @@
 var _ = require('lodash');
 var moment = require('moment');
 
+function cage() {
+  return cage;
+}
+
 function init() {
+  return cage();
+}
+
+module.exports = init;
+
+cage.label = 'Cannula Age';
+cage.pluginType = 'pill-minor';
+
+cage.checkNotifications = function checkNotifications(sbx) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (11)](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/769587957/issues/source?bid=2452379&fileBranchId=2457543#l19)